api/src/main/java/io/strimzi/api/kafka/model/JmxTransSpec.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.JmxTransOutputDefinitionTemplate;
+import io.strimzi.api.kafka.model.template.JmxTransQueryTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Representation for options to be passed into setting up the JmxTrans.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({"outputDefinitions", "queries", "resources"})
+@EqualsAndHashCode
+public class JmxTransSpec implements UnknownPropertyPreserving, Serializable {
+    public static final int DEFAULT_HEALTHCHECK_DELAY = 15;
+    public static final int DEFAULT_HEALTHCHECK_TIMEOUT = 5;
+
+    private static final long serialVersionUID = 1L;
+    protected String image;
+    private List<JmxTransOutputDefinitionTemplate> outputDefinitionTemplates = null;
+    private List<JmxTransQueryTemplate> kafkaQueries = null;
+
+    private ResourceRequirements resources;
+
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("The image to use for the JmxTrans")
+    public String getImage() {
+        return image;
+    }
+
+    public void setImage(String image) {
+        this.image = image;
+    }
+
+
+    @JsonProperty(value = "outputDefinitions", required = true)
```

Comment:
Any reason why not name the property `outputDefinitions` directly? It would make it more readable since you would know what to use int he CR without searching for the annotations.